flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -171,6 +171,8 @@ run_test "ConnectedComponents iterations with high parallelism end-to-end test"
 
 run_test "Dependency shading of table modules test" "$END_TO_END_DIR/test-scripts/test_table_shaded_dependencies.sh"
 
+run_test "RocksDB memory control end-to-end test" "$END_TO_END_DIR/test-scripts/test_rocksdb_state_memory_control.sh" "skip_check_exceptions"
```

Comment:
Tow comments here:
1. I guess we take an optimistic policy here that although in theory there's a chance that this test be unstable, we enable it by default?
2. I believe we also need to add this test into `split_misc.sh` and `split_misc_hadoopfree.sh` under `tools/travis/splits` directory.

Reply:
Will add it to `split_checkpoints.sh` for better balancing across the profiles.